### DISCUSSION TITLE
Want to contribute to 450 DSA questions sheet by Love Babbar using python

### DISCUSSION THREAD

**Discussion by Uchiha-Itachi0** [AkashSingh3031/The-Complete-FAANG-Preparation#199] [General]
Hey, I want to contribute to 450 DSA questions sheet by Love Babbar using python

**Comment by AkashSingh3031** [comment]
Feel  free to contribute
Follow the step-like, the filename with the proper number `01)_Reverse.py` and if you have multiple solutions of the same question then split into multiple parts with the same number like `01_i_)_Reverse.py` & `01_ii_)_Reverse.py`

**Reply by Uchiha-Itachi0** [reply]
Hey man sorry, I just named my file not according to your prescribed rule, and make a pull request. Can I again request by correcting all my mistakes?
